.github/workflows/integration-test.yml

```diff
@@ -23,7 +23,13 @@ jobs:
 
       - name: Package
         run: |
-          ./link-mock-connectors.sh
+          echo Link the mock connectors for integration tests only
+          cd packages/core
+          pnpm link @logto/connector-mock-sms @logto/connector-mock-email @logto/connector-mock-social
```

Comment:
1. I didn't see why `link` suits this scenario in pnpm docs - it doesn't mention the action of adding a dependency in package.json - which is unreliable
2. As we discussed in slack, why don't we just use `lerna add` to add them to the dependency list, what's the specific reason of link and install?

Reply:
Refactored

<img width="2082" alt="image" src="https://user-images.githubusercontent.com/10594507/183345967-92a95b33-d3c5-44b3-9b20-9cb155afdf60.png">

---

2. Intented to use `pnpm link/install` directly and not involve the `lerna` concepts to make the script easier to understand.

Reply:
Refactored. PTAL
```yaml
      - name: Add the mock connectors for integration tests only
        run: |
          unset CI
          unset GITHUB_ACTIONS
          lerna add @logto/connector-mock-sms --scope=@logto/core
          lerna add @logto/connector-mock-email --scope=@logto/core
          lerna add @logto/connector-mock-social --scope=@logto/core
      - name: Package
        run: ./package.sh
```